src/api-service/__app__/onefuzzlib/notifications/ado.py

```diff
@@ -201,6 +201,7 @@ def process(self) -> None:
         for work_item in self.existing_work_items():
             self.update_existing(work_item)
             seen = True
+            break
```

Comment:
Is this a stray change from other WIP?

Doesn't this `break` prevent this function from updating more than one work item?

Reply:
This wasn't meant to be included in this PR.  backing out.